sqlwhat/check_result.py

```diff
@@ -0,0 +1,67 @@
+from pythonwhat.Test import TestFail, Test
+
+def check_result(state, msg="Incorrect result."):
+    stu_res = state.student_result
+    sol_res = state.solution_result
+
+    # empty test
+    test_has_columns(state)
```

Comment:
At some point, you probably want to manually control the message in case no columns have been outputted. Or you're supposed to use `test_has_columns` separately then? The make the 'hierarchical structure' of the check and test functions very clear, it could be useful to make all the messages for the 'subtests' of `check_result` settable through the arguments. Let me know if you disagree!